src/SourceBuild/content/repo-projects/dotnet.proj

```diff
@@ -21,6 +21,8 @@
     <RepositoryReference Include="source-build-reference-packages" />
     <RepositoryReference Include="arcade" />
 
+    <RepositoryReference Include="scenario-tests" />
```

Comment:
This should be listed after `source-build-externals`, right? Since a dependency was declared on it scenario-tests.proj.

Reply:
This should not be considered a toolset project.  Toolset projects are the special bootstrapping tool projects used by everything else.  I think this should be at the very end of the list - you want it to use all built packages and nothing from n-1.  Perhaps this can go in it's own section

Reply:
It lists its dependencies (which are minimal) in the RepositoryReference section of the repo project. This is really what each project should do in order to enable parallelism, and this project would only list the roots of the dependency tree (https://github.com/dotnet/source-build/issues/3608), which would end up as installer and scenario-tests, IIRC.